### ISSUE TITLE
RHEL7 with LinuxFilePath gives unknown file type

### ISSUE THREAD

**Issue by claunia** [qmfrederik/dotnet-packaging#108]
```
error: unpacking of archive failed on file /usr/share/pixmaps/XXXX.svg:5d1752d7: cpio: Unknown file type
error: xxxx: install failed
```

from

```xml
    <Content Include="Assets/xxxxxxx-logo.svg" CopyToPublishDirectory="PreserveNewest" LinuxFileMode="0644">
      <LinuxPath>/usr/share/pixmaps/xxxxxxxx.svg</LinuxPath>
    </Content>
```

**Comment by claunia** [comment]
Seems to happen even if it's not in LinuxPath mmm.

**Comment by claunia** [comment]
If I remove LinuxPath it still does not work, if I change Content to None it works but LinuxPath is ignored.
It works if I remove LinuxFileMode.

**Comment by clemensv** [comment]
Having that same issue on Fedora. LinuxFileMode causes that error.

**Comment by qmfrederik** [comment]
Thanks. I've been doing some work in this area recently. Let me see if I find some time this week to dive into this.

**Comment by qmfrederik** [comment]
The underlying issue was that when you use `LinuxFileMode` to set the lower values of the file mode (e.g. the permissions, such as `755`), the file type (`S_IFREG`, or `10000` in octal) was not set properly. This caused `cpio` to rightfully complain on some older versions of RHEL/CentOS.

This has been fixed with #138 and should be on NuGet soon. I'm also fairly sure this fixes the issue with the sticky bits & friends.
